admin/notifications/views.py

```diff
@@ -44,11 +46,12 @@ def process_duplicate_notifications(request):
 
 @user_passes_test(osf_staff_check)
 def handle_duplicate_notifications(request):
```

Comment:
If the `handle_duplicate_notifications` template / page is being removed, is it safe to remove this and `process_` above as well? Likewise for `admin/notifications/urls.py`.

Reply:
Yes, I have removed the obsolete code.